core/serial_command.py

```diff
@@ -862,7 +815,7 @@ def command_pulse_delay(par):
     us = int((par-ms)*250)
     command = [_cmd_pulse_delay+chr(ms)]
```

Comment:
Done, made sure via search function that all _cmd_ calls have a "p." in front of them.